electron-env

```diff
@@ -9,15 +9,19 @@
 # python-qt and its dependencies will still need to be installed with
 # your package manager.
 
+# System dependencies
+# sudo apt-get install python3-virtualenv python3-pip virtualenv git libsecp256k1-0
+
```

Comment:
Want to include hardware wallet dependencies also? That's a bit of a moving target but I see people in EC slack asking about it occasionally.